start-site/src/main/resources/application.yml

```diff
@@ -1427,6 +1424,181 @@ initializr:
               href: https://googlecloudplatform.github.io/spring-cloud-gcp/reference/html/index.html#cloud-storage
             - rel: guide
               href: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-storage-resource-sample
+    - name: AI
+      content:
+        - name: Spring AI for Azure OpenAI
```

Comment:
I'd call this `Azure OpenAI` rather than `Spring AI for Azure OpenAI`. This aligns with other dependencies like Dynatrace, Grafana, etc, where you get a Micrometer dependency but the name of the entry aligns with the service you're using not the library that you add in order to use it.